tests/integration/targets/x509_certificate-acme/tasks/impl.yml

```
@@ -0,0 +1,46 @@
+---
+- name: "Generate account key"
```

```suggestion
- name: Generate account key
```
I'm not against quotes .. where necessary:D
later they are not used. One approach would look (imo) better:)

---

Fixed in 4c70b27b00a60f72ad246c05e82edc9d9ddbcfd7.